app/tests/bananabot_test.py

```diff
@@ -0,0 +1,500 @@
+import os
+from models.app_state import AppState
+
+# Remove any config file to avoid any interference with tests
+if os.path.exists(AppState._configuration_file):
+    os.remove(AppState._configuration_file)
+
+from main import *
+from models import *
+from utilities import *
+from api.endpoints import *
+from fastapi.testclient import TestClient
+
+import pytest
+import datetime
+
+# Function to remove the config directory after each test
```

Comment:
This function does a little more than what this comment suggests. Consider changing it or removing it given there are comments within the function